index.js

```diff
@@ -571,6 +582,25 @@ function issecure(req, trustProxy) {
 }
 
 /**
+ * Create a new Cookie for this request.
+ *
+ * @param {Object} cookieOptions
+ * @param {Object} req
+ * @param {Boolean} [trustProxy]
+ * @return {Cookie}
+ * @private
+ */
+function createCookie(cookieOptions, req, trustProxy) {
```

Comment:
Can you move this to be alphabetical within all these functions while you're working on the destroy issue?